client/src/components/app.js

```diff
@@ -61,60 +58,45 @@ class App extends React.Component {
     });
   }
 
-  showAboutModal() {
-    Store.dispatch({ type: aboutTypes.ABOUT_DIALOG_OPEN });
-  }
-
-  render() {
-    const { showAbout, session } = this.props;
-
-    let closeAbout = () =>
-      Store.dispatch({ type: aboutTypes.ABOUT_DIALOG_CLOSE });
+  renderContent() {
+    const { session } = this.props;
 
     if (session.error) {
       return (
-        <EmptyState>
+        <EmptyState className="full-page-blank-slate">
           <Alert type="error">
             <span>Login error: {session.errorMessage}</span>
           </Alert>
         </EmptyState>
       );
     }
 
-    if (session.pending || !session.fulfilled) {
+    if (session.pending || !session.fulfilled || (!session.loggedIn && !session.wasLoggedIn)) {
```

Comment:
Optional: looks like Prettier was trying to run on this page, didn't complete its task. 

Was going to ask, we can look at expanding the column width to 100 or 120, right now prettier has a default for 80 columns, you be up for it if we increased it?

Reply:
totally up for it!